video/out/wlbuf_pool.h

```diff
@@ -16,6 +16,7 @@
  */
 
 struct wlbuf_pool_entry;
+struct zwp_linux_buffer_params_v1;
```

Comment:
Good catch, but I think I'd prefer to just include the headers here. This also has `struct vo_wayland_state` tucked away in it which works by luck. i.e.

```
#include "wayland_common.h"
#include "generated/wayland/linux-dmabuf-unstable-v1.h"
```